specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/bigDataPool.json

```diff
@@ -502,6 +513,33 @@
         }
       }
     },
+    "LibraryInfo": {
+      "type": "object",
+      "description": "Library information of a Big Data pool powered by Apache Spark",
+      "title": "Spark Pool library information",
```

Comment:
Information about a library/package created at the workspace level.

Reply:
changed